### PR TITLE
(maint) Tidy some repeated acceptance test code

### PR DESCRIPTION
**Pull request by james-stocks**

### CODE REVIEW

File: acceptance/lib/pxp-agent/config_helper.rb [hunk 1]
```diff
@@ -10,17 +10,10 @@
 PCP_BROKER_PORT = 8142
 
 # SSL directories and files for our standard set of test certs
-SSL_BASE_DIR_WINDOWS = "C:\\\\cygwin64\\\\home\\\\Administrator\\\\test-resources\\\\ssl\\\\"
```
Comment (johnduarte) [inline acceptance/lib/pxp-agent/config_helper.rb:13]: This conflates two operations.
1. Managing getting the test certs on the system
2. Creating the config file.
I would prefer to break these apart. I have amended my PR for this concern

File: acceptance/tests/invalid_ssl_config.rb [hunk 2]
```diff
@@ -4,10 +4,31 @@
...
+# @param logfile path to log file
+# @param expected the string or pattern expected in the log
+def wait_for_log_entry(host, logfile, expected)
+  # If the expected log entry does not appear in log within 30 seconds, then do an
+  # explicit assertion so we get a test fail (not an unhandled error),
+  # and see the log contents in the test failure output
+  begin
+    retry_on(host, "grep '#{expected}' #{logfile}", {:max_retries => 30,
+                                                     :retry_interval => 1})
+  rescue
```
Comment (johnduarte) [inline acceptance/tests/invalid_ssl_config.rb:17]: This is good stuff. Should it be pulled into the helper file?
It probably deserves a separate PR.

Comment (james-stocks) [inline acceptance/tests/invalid_ssl_config.rb:17]: I think a different helper file, as the config_helper was meant to be just helping with pxp-agent.conf

EDIT: I previously suggested putting this in beaker; but retry_on is not a beaker method

**Comment by johnduarte** [comment]
Log file validation implemented in https://github.com/puppetlabs/pxp-agent/pull/266
